cvat-ui/src/components/annotation-page/canvas/views/canvas2d/brush-tools.tsx

```diff
@@ -53,6 +90,26 @@ function BrushTools(): React.ReactPortal | null {
         'polygon-minus': false,
     });
 
+    const setBrushTool = useCallback(() => setCurrentTool('brush'), [setCurrentTool]);
+    const setEraserTool = useCallback(() => {
+        if (!blockedTools.eraser) {
+            setCurrentTool('eraser');
+        }
+    }, [setCurrentTool, blockedTools]);
```

Comment:
Just a little bit safer in some cases (when somebody updates state incorrectly, putting the same object with updated properties)

```suggestion
    }, [setCurrentTool, blockedTools.eraser]);
```

Reply:
The same below